codegenerator/cli/templates/static/codegen/src/eventFetching/ChainManager.res

```diff
@@ -71,42 +54,15 @@ type fetchStateWithData = {
   currentBlockHeight: int,
 }
 
-let isQueueItemEarlierUnorderedBelowReorgThreshold = (
-  ~fetchStatesMap: ChainMap.t<fetchStateWithData>,
-) => (a: multiChainEventComparitor, b: multiChainEventComparitor) => {
-  let isItemBelowReorgThreshold = item => {
-    let data = fetchStatesMap->ChainMap.get(item.chain)
-    item.earliestEvent->FetchState.queueItemIsInReorgThreshold(
-      ~currentBlockHeight=data.currentBlockHeight,
-      ~highestBlockBelowThreshold=data.highestBlockBelowThreshold,
-    )
-  }
-  // The idea here is if we are in undordered multichain mode, always prioritize queue
-  // items that are below the reorg threshold. That way we can register contracts all
-  // the way up to the threshold on all chains before starting.
-  // Similarly we wait till all chains are at their threshold before saving entity history.
```

Comment:
Do we stil have this behaviour? Where all chains only process up to their reorg threshold? It's important that history saving doesn't doesn't start early because it slows down historical sync significantly

Reply:
Yes, we still have the logic. I'll double check why the comment is removed.